src/System.Net.HttpListener/tests/SimpleHttpTest.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Tests
+{
+    [PlatformSpecific(TestPlatforms.Windows)]
+    public class SimpleHttpTest
```

Comment:
Obviously we need many more tests.  What's the plan for that?  As part of this PR?  Immediately after?